api/src/services/report/smsparser.js

```diff
@@ -138,10 +138,6 @@ const lower = str => (str && str.toLowerCase ? str.toLowerCase() : str);
 exports.parseField = (field, raw) => {
   switch (field.type) {
     case 'integer':
-      // keep months integers, not their list value.
-      if (field.validations && field.validations.is_numeric_month === true) {
```

Comment:
no one uses this any more, according to my grep fu of existing projects.

It also was obliterating `list` config which was present in tests (because it ran first and returned out of the block), which is pretty weird and probably not what the test authors wanted.

Reply:
OK git grep does not work how I thought it works (I thought it was always rooted to the repo root not to the directory it's run in), thanks for the super duper important catch!

I'm pretty sure they are still going, let me check with them as to what their intention is, as they *also* use the (undocumented) `list` feature

Reply:
OK it's not a live project anymore, I'm happy dropping this feature out

Reply:
Me too!